.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2.1
+
+executors:
+  clojure:
+    docker:
+      - image: circleci/clojure:lein
+
+jobs:
+  lint:
+    docker:
+      - image: cljkondo/clj-kondo:2022.04.25
+    steps:
+      - checkout
+      - run: clj-kondo --fail-level error --lint .
+
+  test:
+    executor: clojure
+    steps:
+      - checkout
+      - run: lein test-ci
+      - store_test_results:
+          path: target/test-results
+
+  publish:
+    executor: clojure
+    steps:
+      - checkout
+      - run:
+          name: Download dependencies
+          command: lein deps
+      - run:
+          name: Publish to clojars
+          command: lein deploy clojars
```

Comment:
~ You specify `clojars` as `the repository-name`, not sure if leiningen can figure out the repository because `clojars` entry is not set

https://github.com/technomancy/leiningen/blob/master/doc/DEPLOY.md#deployment

Reply:
Yeah - looks like `lein deploy` is correct, based on local testing.